tools/cell_census_builder/scripts/release/census_summary.py

```diff
@@ -2,10 +2,12 @@
 
 import cell_census
 import pandas as pd
-from cell_census_builder.globals import CENSUS_DATA_NAME
+
+from tools.cell_census_builder.globals import CENSUS_DATA_NAME, CENSUS_INFO_NAME
 
 if __name__ == "__main__":
     census_version = sys.argv[1] if len(sys.argv) > 1 else "latest"
+    previous_census_version = sys.argv[2] if len(sys.argv) > 2 else None
 
     census = cell_census.open_soma(census_version=census_version)
```

Comment:
please port to latest API.  I aspire that our internal code to reflects latest & best practices of API use, as it will be demo for others.

In this case, for example, I would base this off of PR #132 and use a context manager.

```python
with cell_census.open_soma() as census:
    ...
```

Reply:
Good point, thanks Bruce!

Reply:
In addition, the `open_soma` will raise if the version is unknown, so I would catch and report that for both open_soma calls

Reply:
ps. alternatively, you could land this on the old API, but it will break soon :-)

Reply:
I believe we need your PR merged to implement this change

Reply:
Yes, I'm blocked by @pablo-gar.  I have just DM'ed him again and will follow-up by mid-day if we don't have motion.